source/includes/fact-verifier-buildIndexes.rst

```diff
@@ -0,0 +1,9 @@
+
+If you start sync with verification enabled and ``buildIndexes``
+set to ``never``, the ``/start`` endpoint returns an error if
+``mongosync`` finds a TTL collection on the source cluster.
```

Comment:
This isn't quite right: the start endpoint won't return an error (we don't know at that point whether or not there are TTL collections), but the migration itself will fail once it sees the TTL collection. (Usually this will happen pretty shortly after /start, but could happen much later, if a TTL index is created on a collection while the migration is in progress.)

Reply:
Fixed